knowledge/style-guide.md

```diff
@@ -15,94 +15,136 @@ parent: Knowledge
 {:toc}
 </details>
 
-This document will present the officially supported Bedrock-Wiki style-guide for addon-creation. The goal of this guide is to promote best-practices while creating addons, and create a consistent format for everyone to follow.
+This document will present the officially supported Bedrock-Wiki style-guide for addon-creation. The goal of this guide is to promote best-practices
+while creating addons and create a consistent format for everyone to follow.
 
-The style-guide is a living, breathing document, which will evolve as addon-creation evolves. Please get in touch if you think something needs to be updated ot changed. 
+The style-guide is a living, breathing document, which will evolve as addon-creation evolves. Please get in touch if you think something needs to be
+updated or changed.
```

Comment:
Another rogue newline?

Reply:
It move part of the line to line 19, my formatter went off :#